assets/dev/js/frontend/elements-handlers-manager.js

```diff
@@ -28,6 +28,7 @@ module.exports = function( $ ) {
 
 	if ( elementorFrontendConfig.experimentalFeatures[ 'nested-elements' ] ) {
 		this.elementsHandlers[ 'nested-tabs.default' ] = () => import( /* webpackChunkName: 'nested-tabs' */ 'elementor/modules/nested-tabs/assets/js/frontend/handlers/nested-tabs' );
+		this.elementsHandlers[ 'nested-accordion.default' ] = () => import( /* webpackChunkName: 'nested-accordion' */ 'elementor/modules/nested-accordion/assets/js/frontend/handlers/nested-accordion' );
```

Comment:
@davseve Should this not be loaded according to it's own experiment? It currently checks `nested-elements`

Reply:
Correct
NIce catch!